### PR TITLE
Add an insecure TLS option

### PR DESCRIPTION
**Pull request by andriisoldatenko**
resolves https://github.com/astronomer/issues/issues/2976

### CODE REVIEW

File: pkg/github/github.go [hunk 1]
```diff
@@ -12,10 +12,6 @@ import (
 	"github.com/pkg/errors"
 )
 
-var (
-	httpclient = httputil.NewHTTPClient()
```
Comment (andriisoldatenko) [inline pkg/github/github.go:16]: unused code.

**Comment by bote795** [comment]
can a test be written for this?

**Comment by andriisoldatenko** [comment]
> can a test be written for this?

i'm not sure, i'll try.

**Comment by andriisoldatenko** [comment]
@bote795 i think it's not possible to write test, since  it's just transport option. Please explain which specific test you want?

**Comment by bote795** [comment]
> @bote795 i think it's not possible to write test, since it's just transport option. Please explain which specific test you want?

Wasn't sure how to either, just wanted to make sure

**Review by bote795 — APPROVED** [review]
lgtm

**Pull request merged**